static/elements/chromedash-gantt.js

```diff
@@ -0,0 +1,206 @@
+import {LitElement, css, html} from 'lit-element';
+import {nothing} from 'lit-html';
+import SHARED_STYLES from '../css/shared.css';
+
+class ChromedashGantt extends LitElement {
+  static get properties() {
+    return {
+      feature: {type: Object},
+      stableMilestone: {type: Number},
+    };
+  }
+
+  constructor() {
+    super();
+    this.feature = {};
+    this.stableMilestone = null;
+    window.csClient.getChannels().then((channels) =>
+      this.stableMilestone = channels['stable'].version);
+  }
+
+  static get styles() {
+    return [
+      SHARED_STYLES,
+      css`
+      :host {
+        width: 200px;
+        margin-top: 8px;
+      }
+      ul {
+        display: block;
+        font-size: 12px;
+        vertical-align: top;
+        padding: 0;
+        margin: var(--content-padding-half);
+      }
+      li {
+        display: inline-block;
+        width: 80px;
+        padding: 4px 8px;
+      }
+      .header li {
+        text-align: center;
+      }
+      .bar {
+        background-repeat: no-repeat;
+        background-size: 210px;
+        background-position: 100px;
+      }
+      .without-ot {
+        background-image: url(/static/img/gantt-bar-without-ot.png);
+      }
+      .with-ot {
+        background-image: url(/static/img/gantt-bar-with-ot.png);
+      }
+      .platform {
+        padding-top: 15px;
+        vertical-align: top;
+      }
+      .diamond {
+        transform: rotate(45deg);
+        width: 36px;
+        height: 36px;
+        text-align: center;
+        margin: auto;
+        border: 2px solid white;
+      }
+      .dev_trial {
+        background: #cfe2f3ff;
+      }
+      .origin_trial {
+        background: #6fa8dcff;
+      }
+      .shipping {
+        background: #0b5394ff;
+        color: white;
+      }
+      .stable {
+        border-color: #444;
+      }
+      .diamond span {
+        font-size: 14px;
+        font-weight: bold;
+        display: table-cell;
+        transform: rotate(-45deg);
+        width: 34px;
+        height: 34px;
+        vertical-align: middle;
+      }
+      .offset_0 { padding-left: 0px; }
+      .offset_1 { padding-left: 8px; }
+      .offset_2 { padding-left: 16px; }
+      .offset_3 { padding-left: 24px; }
+    `];
+  }
+
+  firstPhrase(milestone) {
+    if (milestone <= this.stableMilestone) {
```

Comment:
Haven't seen this `<=` expression before. What does this mean?

Reply:
The listed milestone of the feature is less than the currently available version of Chrome. If the proposed milestone hasn't come, we want to communicate that the listed milestone is the target, but not guaranteed. The use of the returned string is on lines 109, 118, and 127.

Reply:
The `<=` operator in JS is just a plain old `less than` comparison.  It is not related to the arrow operator or anything like that, even though it looks like a backwards one.

Reply:
Yea I was totally confused it with the arrow operator...

Reply:
Also another note that when `this.stableMilestone` is `null`, null will evaluate to 0, which I guess work out in this use case.